opentelemetry/proto/trace/v1/trace_config.proto

```diff
@@ -1,77 +0,0 @@
-// Copyright 2019, OpenTelemetry Authors
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-syntax = "proto3";
-
-package opentelemetry.proto.trace.v1;
-
-option java_multiple_files = true;
-option java_package = "io.opentelemetry.proto.trace.v1";
-option java_outer_classname = "TraceConfigProto";
-
-// Global configuration of the trace service. All fields must be specified, or
-// the default (zero) values will be used for each type.
-message TraceConfig {
```

Comment:
Please don't remove this for the moment. We still have a config capability for the library. I know it is manual for the moment but we can use this as reference to what can be configured.

Reply:
Do you suggest to remove `rpc Config` from `TraceService` but keep trace_config.proto file for future work?

Reply:
Yes

Reply:
Done